collector/gunicorn/collector.yaml

```diff
@@ -0,0 +1,27 @@
+  receivers:
+    statsd:
+    statsd/2:
```

Comment:
I would just remove the `statsd/2` and use `statsd`. You don't need the trailing `/2` unless you're distinguishing multiple instances of the receiver.